packages/experimental/Link/src/Link.types.ts

```diff
@@ -0,0 +1,97 @@
+import { TextTokens, TextProps } from '@fluentui-react-native/text';
+import { IFocusable, IPressableState, IWithPressableEvents, IWithPressableOptions } from '@fluentui-react-native/interactive-hooks';
+import { ViewProps } from 'react-native';
+
+export const linkName = 'Link';
+
+export type ILinkState = IPressableState & {
```

Comment:
naming nit: We're moving away from using "I" in front of interfaces

Reply:
Ah, I see that the problem is the name clash below.

I think we can name this LinkState and the interface below something like "LinkInfo" since the object below has both props and state